### PR TITLE
Added the embedding_lookup to tensorflow frontend and tested

### LINKED ISSUE #17671: embedding_lookup
Add nn function (embedding_lookup) for TensorFlow frontend

### PR DESCRIPTION
**Pull request by eekamal**
fixes #17671

### CODE REVIEW

File: ivy/functional/frontends/tensorflow/nn.py [hunk 1]
```diff
@@ -18,6 +18,11 @@ def _reduce_strides_dilations(dim, stride, dilations):
     return stride, dilations
 
 
+@to_ivy_arrays_and_back
+def embedding_lookup(params, ids, max_norm=None, name=None):
+    return ivy.embedding_lookup(params, ids)
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/tensorflow/nn.py:23]: There is no `embedding_lookup` function in the Ivy API, you'll need to implement a composition for it, here is a guide https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends.html#temporary-compositions

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @eekamal 

Thanks for the PR🙂
Below are needed changes, feel free to reach out if you have any query regarding the comments😊

Happy coding🤞

File: ivy/functional/frontends/tensorflow/nn.py (hunk 1 above)
Comment (eekamal) [reply ivy/functional/frontends/tensorflow/nn.py:23]: I reviewed your stated source and each and every source it pointed to. I found an example but there is no explanation how has that been done. 
Also, how to know if my chosen function is included in the Ivy API?

**Comment by eekamal** [comment]
I reviewed your stated source and each and every source it pointed to. I found an example but there is no explanation how has that been done.
Also, how to know if my chosen function is included in the Ivy API?

File: ivy/functional/frontends/tensorflow/nn.py (hunk 1 above)
Comment (hirwa-nshuti) [reply ivy/functional/frontends/tensorflow/nn.py:23]: If the function is not present in the ivy functional API we use the available ivy functions to implement its behaviour, and suggest if the function should be added to the functional API here https://github.com/unifyai/ivy/issues/3856

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey the compositional frontend functions use other ivy functions and python builtins to implement the behaviour of such functions

**Pull request closed without merging**